### ISSUE TITLE
JPEG view with MPP on ST/STe/Mega

### ISSUE THREAD

**Issue by miniupnp** [atarijookie/ce-atari#54]
A cool new feature would be an image viewer using MPP http://zerkman.sector1.fr/index.php?post/2012/10/08/Atari-ST-Multipalette-Picture-file-format
It necessitate CPU power to build the MPP image that would then be transfered and displayed on the ST

**Comment by miniupnp** [comment]
MPP 1.1 : http://zerkman.sector1.fr/index.php?post/2013/12/10/MPP-version-1.1

**Comment by atarijookie** [comment]
Yeah, having a fast picture viewer for GIF / JPEG / PNG would be nice, I agree ;) Although I guess mp3 playback and possibly video playback would be better ;)
